### ISSUE TITLE
The app doesn't seems to be working

### ISSUE THREAD

**Issue by pedroth** [erichlof/AsteroidPatrol3D#2]
Got this message in console `Mixed Content: The page at 'https://erichlof.github.io/AsteroidPatrol3D/AsteroidPatrol3D.html' was loaded over HTTPS, but requested an insecure script 'http://cdnjs.cloudflare.com/ajax/libs/three.js/r70/three.min.js'.`

PS: great job on the path trancing apps

**Comment by erichlof** [comment]
@pedroth 
Thanks for reporting this - just today I changed the main .html file to load the local copy of three.min.js out of the /js folder rather than trying to pull from the cdn repository. Hopefully that will take care of the errors you were getting. Note that this game still uses a very old r70 version of three.js and probably will not work with the more modern versions of the library.

In all actuality, I haven't really looked at this repo's code in 7 or so years. The path tracing project (thanks for the compliment by the way!) has consumed all of my programming time. If I had the time, I would do a major refactoring and get everything to work with the latest version of three.js, which now has built-in support for game audio, so howler.js is no longer necessary. Howler.js was giving me audio errors the last time I checked, which did not let the game levels proceed correctly.  Again, with a major refactoring, all of these issues would disappear.

Hope it's at least working for you now enough to be somewhat playable. Thanks again! :-)
